source/common/network/utility.h

```diff
@@ -146,11 +146,12 @@ class Utility {
   static Address::InstanceConstSharedPtr getIpv4AnyAddress();
 
   /**
+   * @param v6only a boolean to determine how to set the IPV6_V6ONLY socket option.
    * @return Address::InstanceConstSharedPtr an address that represents the IPv6 wildcard address
    *         (i.e. "::"). Used during binding to indicate that incoming connections to any local
    *         IPv6 address are to be accepted.
    */
-  static Address::InstanceConstSharedPtr getIpv6AnyAddress();
+  static Address::InstanceConstSharedPtr getIpv6AnyAddress(bool v6only = true);
```

Comment:
This is only ever called with v6only = false in a test (that I can see). Perhaps the v6only = false behavior should be handled in Network::Test::getAnyAddress?

Reply:
I'll do that.